apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.Snippets/StorageClientSnippets.cs

```diff
@@ -52,8 +52,8 @@ public void Overview()
             // Sample: Overview
             var client = StorageClient.Create();
 
-            // Create a bucket
-            var bucketName = Guid.NewGuid().ToString(); // must be globally unique
+            // Create a bucket with a globally unique name
+            var bucketName = Guid.NewGuid().ToString();
```

Comment:
IdGenerator.FromGuid()?

Reply:
Nope, this one's in a sample, so I don't want to refer to any extra methods.